packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,248 @@
+
+```jsx
+<Dropdown selectionType="multiple">
+  <SelectInput label="Select Emails" />
+  <BottomSheet>
+    <BottomSheetHeader title="Do you want this report in an email?">
+      <AutoComplete label="Add receiver's email addresses" />
+    </BottomSheetHeader>
+    <BottomSheetBody>
+      <ActionList>
+        <ActionListItem title="Mumbai" value="mumbai" />
+        <ActionListItem title="Pune" value="pune" />
+        <ActionListItem title="Hyderabad" value="hyderabad" />
+      </ActionList>
+    </BottomSheetBody>
+  </BottomSheet>
+</Dropdown>
+```
+
+### With Tags Outside
+
+When possible selections are more than 11, it is advised to put tags outside of the Input.
+
+> **Note**
+>
+> This screenshot is outdated. The input will show "Status (13 Selected)" instead of just "Status". Will update this soon.
+
+<img width="700" alt="image" src="https://github.com/razorpay/blade/assets/30949385/8906eb6c-fae0-4e99-bd3f-d7e0edc60537">
+
+**Code**
+
+```jsx
+const [selectedItems, setSelectedItems] = React.useState([]);
+
+// ...
+
+<AutoComplete
+  label="Status"
+  labelPosition="inside-input"
+  maxRows="single"
+  value={selectedItems}
+  onChange={({ values }) => setSelectedItems(values)}
+/>
+
+
+// Somewhere outside in your UI
+<Box>
+  {selectedItems.map((selectedItemName, index) =>
+    <Tag
+      onDismiss={() => removeItem(index)}
+    >
+      {selectedItemName}
+    </Tag>
+  )}
+</Box>;
+```
+
+### With Different Label Positions
+
+While things are flawless when you give visible label (highly recommended), If there are scenarios where you are not able to give label to inputs (questionable but ok), you can use `labelPosition: 'inside-input'` prop.
+
+| **Code**                                                       | **Description**       | **Single Select**                                                                                                | **Multi Select**                                                                                                 |
+| -------------------------------------------------------------- | --------------------- | ---------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| `<AutoComplete label="Status" /> `                             | With Label Present    | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/0c559daa-5cdc-4d71-9221-6073b7969b1b" /> | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/9314fd11-96b7-46ef-963f-283b393cb9fd" /> |
+| `<AutoComplete label="Status" labelPosition="inside-input" />` | Without Label Present | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/2cb1d659-4a98-4f63-96d8-204ead4a681f" /> | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/0b219c4c-2df2-4cfe-a32d-4c02809ab440" /> |
```

Comment:
when we provide label position inside then label is what we show as prefix always? shouldn't that be configurable by user for better copywriting content?

Reply:
> when we provide label position inside then label is what we show as prefix always? shouldn't that be configurable by user for better copywriting content?

I am not sure if people will spend time getting it right. It might end up becoming inconsistent where 2 dropdowns on same page have 2 different ways of telling what is selected and what is label.

Reply:
> for inside-input and multiselect why don't we do it like we do for label outside thing? show Status: 1 tag and +6more?

1. It takes more space which we won't have in smaller inputs.
2. Its a fairly rare usecase so we just wanted to do the easiest thing possible rather than spending extra time on this. E.g. Our number of tags to show and `+x more` calculations will need some extra fixing when we add another text element in the input.
3. Even when it is used, it is mostly expected to be used in headers where they are probably showing tags outside. In forms, they should just show label.